pandas/core/dtypes/common.py

```diff
@@ -90,9 +90,11 @@ def ensure_categorical(arr):
 def ensure_int64_or_float64(arr, copy=False):
     """
     Ensure that an dtype array of some integer dtype
-    has an int64 dtype if possible
+    has an int64 dtype if possible.
```

Comment:
can you move these comments in to a Notes section.

Reply:
done